data/service/user.go

```diff
@@ -159,9 +159,13 @@ func (us *UserService) LoginUser(username, password string) (string, model.Table
 
 		groups, err := us.LdapClient.GetGroupsOfUser(username)
 		if err != nil {
-			logrus.Error("Couldn't get any group for user: ", username)
+			logrus.Error("Couldn't get any group for user ", username, ": ", err)
 		} else {
-			if heputils.ElementExists(groups, us.LdapClient.AdminGroup) {
+			logrus.Debug("Found groups for user ", username, ": ", groups)
+			// ElementExists returns true if the given slice is empty, so we explicitly check that here
```

Comment:
Wouldn't really know if this is intentional or not - but since `ElementExists` is being used across the project, I didn't want to change its behaviour.